src/ui/checkbox/Checkbox.tsx

```diff
@@ -16,28 +14,18 @@ namespace Checkbox {
   }
 }
 
-const checkboxClasses = { root: csx.checkbox, checked: csx.checked };
-
 const Checkbox = ({ label, value, variant = 'default', dataIdx, onChange }: Checkbox.Props) => {
   return (
-    <FormControlLabel
-      label={label}
-      classes={{
-        root: `${csx.checkboxLabel} ${csx[variant]}`
-      }}
-      control={
-        <MuiCheckbox
-          checked={value}
-          onChange={onChange}
-          classes={checkboxClasses}
-          inputProps={
-            {
-              'data-idx': dataIdx
-            } as any
-          }
-        />
-      }
-    />
+    <label className={`${csx.checkboxLabel} ${csx[variant]}`}>
+      <input
+        type="checkbox"
+        checked={value}
+        className={`${csx.checkbox} ${value ? csx.checked : ''}`}
+        onChange={onChange}
```

Comment:
You should create method handleChange - which takes as deps onChange and triggers onChange with dedicated parameters. For material UI value argument was a bool negation - please check this